client/my-doggo.js

```diff
@@ -48,4 +47,92 @@ async function loadDoggos() {
 	}
 }
 
+const logOutBtn = document.querySelector('#logOut')
+
+logOutBtn.addEventListener('click', async e => {
+	let response = await fetch('/logout')
+	console.log(response)
+
+	if (response.ok) {
+		window.location.replace(response.url)
+	} else {
+		alert('HTTP-Error: ' + response.status)
+	}
+})
+
+const deleteModalBody = document.querySelector('.deleteModalBody')
+
+const deleteRequest = async (data = {}) => {
+	const response = await fetch('/users/delete-doggo', {
+		headers: {
+			'Content-Type': 'application/json'
+		},
+		method: 'DELETE',
+		body: JSON.stringify(data)
+	})
+
+	let status = response.status
+	let res = await response.json()
+
+	return { status, res }
+}
+
 loadDoggos()
+	.then(result => {
+		// Add event listener on every doggo card's edit button
+		const editDoggo = document.querySelectorAll('.editDog')
+
+		editDoggo.forEach(doggo =>
+			doggo.addEventListener('click', async e => {
+				const doggoName =
+					e.target.parentElement.parentElement.nextElementSibling
```

Comment:
There has to be a better way to find this element

Maybe using an id/class and finding it that way using a selector?

Reply:
I was thinking the same, like I was thinking of just adding an id to the edit button with the dogName, but a dog name can have a space in it, which means I can't use it for class/id